opendbc/can/common.h

```diff
@@ -44,7 +44,7 @@ unsigned int pedal_checksum(uint32_t address, const Signal &sig, const std::vect
   } while (false)
 
 inline bool endswith(const std::string& str, const char* suffix) {
-  return str.find(suffix, 0) == (str.length() - strlen(suffix));
+  return str.find(suffix, str.length() - strlen(suffix)) != std::string::npos;
```

Comment:
https://cplusplus.com/reference/string/string/find/ says that a pos input bigger than the string is defined to return no match. Negative values are wrapped around to large positive values, so no undefined behavior here